src/features/create-quiz/store/CreateQuizStore.tsx

```diff
@@ -3,50 +3,63 @@ import { Question } from "../create-quiz-second-card/create-quiz-second-card";
 import uuid from "react-uuid";
 
 export interface QuestionItem {
-  id: number;
+  id: string;
   question: Question | null;
   completed: boolean;
 }
 
-interface SelectedQuestion {
-  question: QuestionItem | null;
-  isSelected: boolean;
-}
-
 export class QuizStoreImpl {
   questions: QuestionItem[] = [];
-  selectedQuestion: SelectedQuestion | null = null;
+  selectedQuestionID?: string;
 
   constructor() {
     makeAutoObservable(this, {}, { autoBind: true });
   }
 
-  addQuestion(question: Question | null) {
+  addQuestion(question: Question | null, isCorrectResponse: number | null) {
```

Comment:
The parameter `isCorrectResponse` seems to be a boolean from its notation, but I see it is not. I suggest changing it to something like `correctResponseIndex`.